front-end/src/screens/Post/EditablePostContent.tsx

```diff
@@ -127,15 +176,19 @@ const EditablePostContent = ({ className, onReply, post, refetch }: Props) => {
 export default styled(EditablePostContent)`
 	margin: 2rem 0;
 
+	.unsubscribe {
+		color: red!important;
```

Comment:
I like the idea to have it in a different color. This red is a little too red though. I used the same red as the background of the notifications. @erler feel free to change.
```suggestion
		color: #FF474E !important;
```